core/corehttp/gateway_handler.go

```diff
@@ -125,6 +125,10 @@ func (i *gatewayHandler) getOrHeadHandler(w http.ResponseWriter, r *http.Request
 	i.addUserHeaders(w) // ok, _now_ write user's headers.
 	w.Header().Set("X-IPFS-Path", urlPath)
 
+	//robots-header to prevent search-engines from indexing the gateway because of the duplicate-content problematic when makeing the gateway public
+	w.Header().Set("X-Robots-Tag", "noindex")
```

Comment:
Btw, you can set this now with the HTTPHeaders options -- see `ipfs daemon --help`